pkg/apis/lighthouse.submariner.io/v2alpha1/types.go

```diff
@@ -89,7 +89,7 @@ type ServiceImport struct {
 type ServiceImportType string
 
 const (
-	// Services are only accessible via their supercluster IP.
+	// Services are only accessible via their clusterset IP.
```

Comment:
May need to change this too. I've sent mail to sig for clarification.

Reply:
Good point, I’ll switch the PR back to draft until we get confirmation from the SIG.